content/SCALE/SCALETutorials/DataProtection/Replication/ReplicationWithEncryptionSCALE.md

```diff
@@ -0,0 +1,142 @@
+      If your source is the local TrueNAS SCALE system, you must select **On a Different System** from the **Destination Location** dropdown list to do remote replication. 
+      
+      TrueNAS shows the number snapshots available for replication.
+    
+    b. Select an existing SSH connection to the remote system.
+       Select **Create New** to open the **[New SSH Connection](#configure-a-new-ssh-connection)** configuration screen.
+    
+    c. Browse to the source pool/dataset(s), then click on the dataset(s) to populate the **Source** with the path. 
+       You can select multiple sources or manually type the names into the **Source** field. 
+       Selecting **Recursive** replicates all snapshots contained within the selected source dataset snapshots.
+
+    d. Repeat to populate the **Destination** field. 
+       You cannot use zvols as a remote replication destination. 
+       Add a **/*datasetname*** to the end of the destination path to create a new dataset in that location.
+    
+    e. (Optional) Select **Encryption** to add a [second layer of encryption](#adding-additional-encryption) over the already encrypted dataset.
+         
+    f. Select **Use Sudo for ZFS Commands**. A **Sudo Enabled** dialog displays to allow you to select this option, or you can select it on the wizard screen.
+    
+    ![UseSudoForZFSCommandsDialog](/images/SCALE/22.12/UseSudoForZFSCommandsDialog.png "Select Use Sudo for ZFS Commands")
+    
+       This option only displays when logged in as the admin user.
+       If not selected you need to issue the cli `zfs allow` command in Shell on the remote system. 
+
+    g. Select **Replicate Custom Snapshots**, then accept the default value in **Naming Schema**. 
+       Remote sources require entering a snapshot naming schema to identify the snapshots to replicate. 
+       A naming schema is a pattern of naming custom snapshots you want to replicate. 
+       If you want to change the default schema, enter the name and [strftime(3)](https://man7.org/linux/man-pages/man3/strftime.3.html) %Y, %m, %d, %H, and %M strings that match the snapshots to include in the replication. 
+       Separate entries by pressing <kbd>Enter</kbd>. The number of snapshots matching the patterns display.
+    
+    ![RemoteReplicateSnapshotAndNameSchema](/images/SCALE/22.12/RemoteReplicateSnapshotAndNameSchema.png "Replicate Custom Snapshot and Naming Schema")
+
+   h. (Optional) Enter a name for the snapshot in **Task Name**. 
+      SCALE populates this field with the default name using the source and destination paths separated by a hyphen, but this default can make locating the snapshot in destination dataset a challenge. 
+      To make it easier to find the snapshot, give it name easy for you to identify. For example, a replicated task named *dailyfull* for a full file system snapshot taken daily. 
```

Comment:
Missing a word: 'give it name' to 'give it a name'
Also, 'name easy for you to identify' to 'name for you to easily identify'